src/components/Registration/NaicsResult.vue

```diff
@@ -1,41 +1,95 @@
 <template>
-  <div class="naics-result">
-    <!-- *** TODO: format this as per comps -->
-    <v-btn text @click="emitClick(result)">
-      {{result.classTitle}}
-    </v-btn>
-  </div>
+  <v-row no-gutters class="naics-result" v-if="!!result" @click="emitClick(result)">
+    <v-col cols="2">
+      <div class="result-code">{{result.code}}</div>
+    </v-col>
+
+    <v-col cols="10">
+      <label class="result-class-title">{{result.classTitle}}</label>
+
+      <div class="result-class-definition">{{result.classDefinition}}</div>
+
+      <template v-if="elements.length > 0">
+        <div class="sample-activities">Sample activities:</div>
+        <ul>
+          <template v-for="(element, index) in elements">
+            <li v-if="showMore || (index < 5)" :key="index">
+              {{capitalize(element.elementDescription)}}
+            </li>
+          </template>
+        </ul>
+
+        <div v-if="!showMore && (elements.length >= 5)" class="show-more" @click.stop="showMore = true">
```

Comment:
Here I use the "stop" modifier to prevent the event from bubbling.

If you want to see this in action, you'll have to download this PR locally, since the Show More feature is currently disabled.